Changes

```diff
@@ -54,6 +54,13 @@ Working version
 - #9582: Add Array.{find_opt,find_map,split,combine}.
   (Nicolás Ojeda Bär, review by Daniel Bünzli and Gabriel Scherer)
 
+- #XXXXX: Add support for Unix domain sockets on Windows.
+  (Antonin Décimo, review by David Allsopp)
+
+- #XXXXX: Emulate socketpair for Unix domain sockets of streaming type
+  on Windows.
+  (Antonin Décimo, review by David Allsopp)
+
```

Comment:
I'd merge these into one entry when you update it: something like "Add support for Unix domain sockets on Windows and use them to emulate Unix.socketpair (only available on Windows 1803+)"